MAUI/Shimmer/built-in-views.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: About .NET MAUI Shimmer control | Syncfusion
+description: Learn here about the Built-in types support of Syncfusion .NET MAUI Shimmer (SfShimmer) control and more.
+platform: maui
+control: SfShimmer
+documentation: ug
+---
+
+# Built-in Views of .NET MAUI Shimmer (SfShimmer)
+
+## Built-in types
+
+Syncfusion .NET MAUI Shimmer (SfShimmer) control provides seven different built-in shimmer view types.
+
+* `CirclePersona`
+* `SquarePersona`
+* `Profile`
+* `Article`
+* `Video`
+* `Feed`
+* `Shopping`
+
+The built-in shimmer types can be used by setting the `Type` of `SfShimmer`. By default, `CirclePersona` is initially rendered.
```

Comment:
The built-in shimmer types can be used by setting the `Type` of `SfShimmer.` By default, the `CirclePersona` is initially rendered.

Reply:
Contents modified